### PR TITLE
Ipfs work

### PR DESCRIPTION
**Pull request by UsamaSaleem14**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -24,7 +25,7 @@
     "web-vitals": "^2.1.0"
   },
   "scripts": {
-    "start": "react-scripts start",
+    "start": "set \"GENERATE_SOURCEMAP=false\" && react-scripts start",
```
Comment (TBomberman) [inline package.json:28]: What is GENERATE_SOURCEMAP for? Is this the right place to set the variable?

File: package.json [hunk 2]
```diff
@@ -54,5 +55,8 @@
     "eslint-plugin-react-hooks": "^4.6.0",
     "prettier": "2.8.1",
     "tailwindcss": "^3.2.4"
+  },
+  "resolutions": {
```
Comment (TBomberman) [inline package.json]: There are no values in this element. Can it be removed?

File: src/components/IpfsUpload/index.tsx [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+  };
+  return (
+    <>
+      <input
+        type="text"
+        className={"border border-sky-500 mt-2"}
+        onChange={getData}
+        placeholder={"Enter data"}
+      />
+      <button onClick={dataFunc} className={"border border-sky-500 mt-2"}>
```
Comment (TBomberman) [inline src/components/IpfsUpload/index.tsx:41]: This component is just to test uploading things to IPFS?

Comment (UsamaSaleem14) [reply src/components/IpfsUpload/index.tsx:41]: Yes

File: package.json (hunk 1 above)
Comment (UsamaSaleem14) [reply package.json:28]: It is used to remove a warning that I get in my console when I start the project. When I start the react app it generated the source map files so some packages are causing an issue of which I have added a screenshot. When we do the above solution it works without any warning in the console.

image

File: package.json (hunk 2 above)
Comment (UsamaSaleem14) [reply package.json]: Done

**Comment by TBomberman** [comment]
Please add the env instructions the app id in the readme.

**Comment by UsamaSaleem14** [comment]
> Please add the env instructions the app id in the readme.

Done

**Pull request merged**